plugins/org.jkiss.dbeaver.ext.dsql/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jkiss.dbeaver</groupId>
+    <artifactId>plugins</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+     <relativePath>../</relativePath>
+  </parent>
+  <artifactId>org.jkiss.dbeaver.ext.dsql</artifactId>
+  <version>1.0.151-SNAPSHOT</version>
+  <packaging>eclipse-plugin</packaging>
+
+  <properties>
+    <aws.sdk.version>2.31.78</aws.sdk.version>
+  </properties>
+
+  <build>
```

Comment:
Thank you for the contribution!

But unfortunately, we cannot include it in dbeaver build.
It adds dependency on certain version of AWS SDK (which is also quite heavy, +10mb).
And we already have AWS in Pro products - it may (and will) conflict because of different SDK versions.
IAM authentication is also present in Pro products and it will clash with this one.

If you still want to add this extension to community DBeaver then you can create a separate Markeplace extension. 
Similarly to other dbeaver extensions like git, office or external ones:
https://marketplace.eclipse.org/content/explain-postgresql or https://marketplace.eclipse.org/content/dbeaver-vault